src/libpriv/rpmostree-util.c

```diff
@@ -609,6 +609,16 @@ rpmostree_migrate_pkgcache_repo (OstreeRepo   *repo,
   return TRUE;
 }
 
+char*
+rpmostree_get_deployment_root (OstreeSysroot     *sysroot,
```

Comment:
At some point we should try to drive some fd-relative stuff into libdnf...maybe someday.